.github/workflows/consistency-functional.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       max-parallel: 4
       matrix:
         python-version: [3.7]
-        ansible-version: [2.9, 3.4, 4.5]
+        ansible-version: [2.9, 4.5]
```

Comment:
In spirit yes, let's also rename '4.5' to 'latest' so that we don't have to keep renaming files and editing many places whenever we bump the Ansible there to something newer?

Reply:
We'll also want to make the same matrix change for the functional test below on line 101.